### DISCUSSION TITLE
30.10.2023 update

### DISCUSSION THREAD

**Discussion by mtzortzi** [mtzortzi/SpMV_collab#2] [Show and tell]
### Update Sum Up Meeting

1. ReadMe
2. Validation of the true matrices
3. plots
4. decision tree
5. SVR
6. feature importance
7. CNNs

**Comment by avinot244** [comment]
ReadMe content finished. I just need to work a bit on the page layout. Also the Dataset Info section is not done yet

**Reply by mtzortzi** [reply]
Can we link the .py files with the actual code on ReadMe? Like I made it with the .csv in the Dataset section

**Reply by avinot244** [reply]
yeah sure I'll let you handle it I'm not that good on markdown files ^^'

**Comment by avinot244** [comment]
Validation of the true matrices and plots are done

**Comment by avinot244** [comment]
Plots are done

**Comment by avinot244** [comment]
decision trees and svr are done
